### ISSUE TITLE
Display mentor options for appearing in search results and online mentorship in chapter selection modal

### ISSUE THREAD

**Issue by dboyer** [Iridescent-CM/technovation-app#5084]
On the screen where mentors select a chapter, they should also view the options to do two things:

- A check box to "Allow teams to find you in search results and invite you to join"
- A check box to "Indicate to teams that you can be an online, remote mentor"

<img width="607" alt="Screen Shot 2024-10-15 at 9 56 08 PM" src="https://github.com/user-attachments/assets/1919f76a-50d8-4b55-9a0c-b59970e5cd1c">

These two fields are on the mentor profile but VET would like to make them more prominent by putting them in the chapter selection process. They should also remain on the mentor profile page where they can be edited.

**Comment by dboyer** [comment]
Oh good question. I think it's fine to not pre-check the values. I'll add it to the notes for dev priorities.

**Comment by dboyer** [comment]
Tested on preview and confirmed:

- After registering as a mentor, I see the two boxes on the chapter selection page

A couple things @shaun-technovation 

1. It looks like these two options are no longer on the mentor profile so the mentor cannot edit the selections later. They were previously located on the basic tab of the mentor profile (/mentor/profile#!basic). Could we put them back there so the mentor can update them later if needed?

2. I selected to appear in search results for a few mentors but my mentor profile did not appear as an option in search results when I logged in as a student searching for a mentor. Specifically:

- Registered three mentors and selected to appear in search results (/admin/participants/1674#/, 1673, 1672) 
- Made sure all three mentors were onboarded (training, consent waiver, background check)
- Logged in as a student with a team (/admin/participants/1675#/)
- Went to the mentor search page (/student/mentor_searches/new), set the location to anywhere, and searched for teams
- Saw no mentors as search results
